### ISSUE TITLE
Beware the 600 hour bug, possible Niantic security measure

### ISSUE THREAD

**Issue by pill0ws** [mchristopher/PokemonGo-DesktopMap#883]
I dont know much about programming but I have aan errie suspicion that the new bug related to long duration spawn times could be a new security measure to identify scanners. These pokemon do not exist in the game but they show up in a scan update. It really seems like "Dummy" spawns. Could this potentially be a new detection method employed by Niantic? My PTC account is froze, I logged into the site and couldnt even change any account settings. I logged in with a seperate account and had no problems. I logged in to the app with a google account and it seems to work fine but is picking up these endless spawn timer pokemon that are not really in the game world. Does anyone with programming knowledge know of anything like this? like maybe some tripwire type thing? I am probably not going to use the app at all and see how this banwave plays out but I am curious if this is a coincidence or if the dummy pokemon are related to new tactics

**Comment by Regulator69** [comment]
no just a glitch in the system or bug or whatever just saying, my ekans said 591 hours and refreshed the page and it said the right time at 8 and half minutes and i went and got him so he was there..  just if you dont refresh the page you will never know for sure when that pokemon despawned cuz it wont til that timer runs out so they could be gone long before that timers done of course.. so yea it would be a decent idea, but you would have to do a ton not just a one or two at a time which is the case at the time.. and if i refreshed and they werent still there everytime but yea just have to double check em if they are important by just refreshing..

**Comment by Hrafn81** [comment]
First, fastpokemap solves this by having a "max" despawn of 15 minutes and 30 seconds.

Secondly, I'm using the Windows installer, refresh doesn't remove anything. I need to manually remove them. And at times, that's a lot.

Not sure if it's a "bug" since everyone started noticing them at the same time. My money is on Niantic causing this.

**Comment by pill0ws** [comment]
some are showing up as already despawned though, like the never existed to begin with

**Comment by anon2019** [comment]
just delete the pokemon folder under appdata/local/ fixes it for me 
all good after that

**Comment by anon2019** [comment]
I delete 
C:\Users\users\AppData\Local\pokemon
and
C:\Users\users\AppData\Local\SquirrelTemp
but i noticed my info stayed in the login boxes and such and so did the 600 hr pokemon

Then delete
C:\Users\user\AppData\Roaming\pokemon-go-map
and this cleared it up i don't think it has anything to do with nantic or if it does clearing all your stuff works fine
do this yeah baby.

**Comment by Hrafn81** [comment]
Testing with four PTC accounts, one of them the old one. Seems so far only the old one is getting new 600hour spawns. Needs more testing. (Opened client four times in Windows using shift, all clients sending to same API. Not that it matters for this testing. The four accounts are roaming different areas.) :)

Also, right now I don't want to delete all cache, reading the "full stats" to look for returning pokemons. Will try that later.

**Comment by pill0ws** [comment]
is anyone else getting the exact same disappear time on all of the "sticky spawns". Mine all disappeared at 09:56:22 this morning and have 587h 42m left as of this message

**Comment by Hrafn81** [comment]
So opened four clients in Windows, using four different PTC accounts. One of them the old one.
All scanning four different areas, so far only the old account has found new 600hour spawns.
edit: One new such spawn right outside the grid of the old account, into the area of one of the new accounts. Might be my fault for not making one of those grids smalled.

Needs more testing, going to let computer work while I sleep.

Also, will try to delete the temp folders later. Using "full stats" to track returning mons :)

**Comment by Hrafn81** [comment]
Ok, new info from me. It's not related to the account. I'm already getting this on the new accounts and they are all on the same expiration time, like pill0ws mentioned. 4:56:22pm at +1 timezone in Europe. (summertime, so works like +2)

**Comment by pill0ws** [comment]
I am central Time zone. Hang on, I'll check the hour gap

**Comment by dhyde79** [comment]
having the same issue....I went in and totally cleaned everything, which of course required a full reinstall, they were all gone for a while, now they're back again....

**Comment by Mirx** [comment]
https://github.com/PokemonGoMap/PokemonGo-Map/issues/566

**Comment by Hrafn81** [comment]
https://github.com/PokemonGoMap/PokemonGo-Map/commit/0f19c0a85b421a83c5c0d50c6fd39d59c1040f2c

Manually edited my models.py and testing it now.

**Comment by pill0ws** [comment]
So, maybe I've been reading too many conspiracy theories lately. I went out last night and consistently saw this bug, however I manually removed them. Often the pokemon actually was spawned when the map showed a goofy result. Interestingly enough, they also seemed to always contain the same species but I was in an area with very little variety to begin with (riverbank with exclusively psyduck, magikarp, and slowmo. occasionally a dratini, squirtle or tentacool. So its possible I observed the spawn points only contain one species the entire night because the area tended toward only 3. However the magikarp spawns with the crazy timer, always had Magikarp and the psyducks always had psyducks. All other spawn points seemed to alternate to some degree. This was just an observation and would need to be tested other places. I definately feel like the glitch is resultant from some change that Niantic made to spawn points in general and not some shady security setup

**Comment by Hrafn81** [comment]
Most likely it's a 30 minute spawn, but I never had issues prior to this. I believe it used to be a negative value that showed as positive suddenly and that flipped the map. Trying to fix this, I ended up with a stuck "loading" screen after I exited and deleted all cache. Checked accounts and IP, no ban. Made new PTC accounts, same problem. In the end I downloaded the files from development (pogom folder) and pasted them into the windows folders and it all started working again. Yes, I tried deleting everything and reinstalling first.

Anyway, it works for me now. Only bug now is that I can't see "scanned" locations but I can live with that. Using 5 accounts to scan 5 tiny areas, it updates pretty quick.

**Comment by eluarca1971** [comment]
Hi, as a temporal solution: uninstall Pokemon go Live map. Don't reinstall yet, go to: C:\Users\AppData\Roaming\ there delete the Folder: "Pokemon Go Live Map" there is where the app stores the Access data and the log of pokemon located in the time. once that, reinstall the app and input your acces data (Google Key and coordinates) Note: BEFORE the uninstall be sure that you have write down at least your Google Key) I know that this is a not programming soultion, but at least that give you tme to find the real solution. I hope that this can help you.

**Comment by jecates** [comment]
A co-worker was running the map while I was at his desk, and he hit on a Grimer nearby, that showed 500+ hrs. I went back to my desk, and ran the app in the same area, and the Grimer showed up for me with ~11 minutes. We walked to where it showed for both of us, and it was actually there for both our mobile clients. We were running the scanner using our actual accounts, so the scanner and the mobile client should be getting the same data.

I do know a *little* bit about programming (I'm an analyst on an iOS dev team), and I'm not sure how this could provide Niantic any information about using 3rd party clients. It's a value they are sending from their servers to the client. Unless the mobile client sends back a "timer too long" message, they don't really know what the client does with it. Since his mobile client picked up the poke, even though the scanner client didn't, I think it's extremely unlikely they are actually getting a message back about the timer - they would have to get a message back AND somehow provide the client with information that the poke actually SHOULD be there, along with when it expires. Trying to build this information into an actual useful data element seems very inelegant, to say the least. I can't imagine a professional developer coming up with such a solution.

If it ONLY showed up on the scanner, I could see it being a "trap", but only if the user interacts with it. They could build into the mobile client to ignore pokes with timers > 1 hr, so it wouldn't show up, and it might go unnoticed by other developers. Then, botters who throw balls at these 600 hr guys would clearly be using a 3rd party client.
